lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+const TraceProcessor = require('../../lib/tracehouse/trace-processor.js');
+
+const LH_ATTRIBUTE_MARKER = 'lhtemp';
+
+/**
+ * @fileoverview
+ * This gatherer find the Largest Contentful Paint element identified in the trace. Since the trace only has the nodeId of the element,
```

Comment:
```suggestion
 * This gatherer identifies the Largest Contentful Paint element from the trace. Since the trace only has the nodeId of the element,
```